test/jdk/java/net/httpclient/ManyRequests.java

```diff
@@ -260,13 +271,13 @@ static void test(HttpsServer server, HttpClient client) throws Exception {
                 done = true;
             } catch (CompletionException e) {
                 if (!Platform.isWindows()) throw e;
-                if (LIMIT.get() < REQUESTS) throw e;
+                if (LIMIT.get() < MAX_LIMIT) throw e;
```

Comment:
Previously, `LIMIT.get()` was being compared against `MAX_COUNT`, because `REQUEST` was initialized (as `final`) to that value. Now it's being compared to `MAX_LIMIT`. Is this intentional?

Reply:
What it says here is that if we already reduce the LIMIT once and it fails again we should fail.